src/client/components/autocomplete2/index.tsx

```diff
@@ -79,6 +80,12 @@ export class Autocomplete extends Component<
       remote: {
         url,
         filter: filter || returnItems,
```

Comment:
This is just an additional safeguard to ensure the header is being set with validated user tokens?

Reply:
Yeah, we need to manually set the access token for Bloodhound, before we were running un-authenticated requests. (the end game is showing a mix of published/unpublished content on the autocomplete, but that is not yet implemented here).

Reply:
Nice!